package.json

```diff
@@ -3,7 +3,7 @@
   "description": "supervisor and monitor for node.js applications",
   "author": "Sam Roberts <sam@strongloop.com>",
   "license": "Artistic-2.0",
-  "version": "3.3.1",
+  "version": "4.0.0",
```

Comment:
change to `4.0.0-1` so it has a pre-release number (so it won't be considered as a valid 4.x release yet)